x-pack/plugin/esql/qa/testFixtures/src/main/resources/union_types.csv-spec

```diff
@@ -1720,6 +1737,34 @@ id:integer | name:keyword | count:long
 14         | mmmmm        | 2
 ;
 
+shortIntegerWideningFork
+required_capability: union_types
+required_capability: fork_union_types
+required_capability: casting_operator
+required_capability: union_types_numeric_widening
+required_capability: fork_v9
+
+FROM apps, apps_short
+| EVAL x = id::integer
+| FORK (WHERE true) (WHERE true)
+| DROP _fork
```

Comment:
can we do something with `x` after FORK? for example adding `EVAL x = x + 1` or something similar?
just to validate that the value can be used.